src/lib/y2users/linux/writer.rb

```diff
@@ -124,15 +130,38 @@ def write
       CHPASSWD = "/usr/sbin/chpasswd".freeze
       private_constant :CHPASSWD
 
+      # Whether given user does not exist yet
+      #
+      # FIXME: choose a better criteria to identify a new user. An internal Y2User::User id?
```

Comment:
I think internal ID is not good as it can have problem with some operations. Here the biggest topic is to distinguish between two cases:

1. User is deleted and new one is created with same ID -> new home, so old user data is removed
2. rename of user so same ID -> keep old home, do not remove any data!

change uid of user is different topic and very tricky as it means to traverse whole disk and chown from old one to new one...probably we should not support it.